tests/v1/Billing_Pant_Test.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from api.db.database import Base
+from api.utils.billing import create_billing_plan
+from api.v1.models import BillingPlan
```

Comment:
correct the typo error in the file naming

billing_plan_test.py would be better